include/openthread/platform/radio.h

```diff
@@ -276,10 +279,15 @@ typedef struct otRadioFrame
              * Note: `mTxDelayBaseTime` + `mTxDelay` SHALL point to the point in
              * time when the end of the SFD will be present at the local
              * antenna, relative to the local radio clock.
+             *
+             * This field together with mTxDelayBaseTime should be orthogonal to mCsmaCaEnabled. If this field is
```

Comment:
The term "orthogonal" may not be immediately clear.

I understand it to mean that when `mTxDelay(Base)Time` is non-zero, CCA should always be performed (regardless of the `mCsmaCaEnabled` value), and CSMA backoffs should never be done (`mMaxCsmaBackoffs` should be ignored). Is this correct?

My suggestion is to keep the rules simple:

  - `mCsmaCaEnabled` should always control whether the radio platform performs CCA (listen-before-talk or just send).
  - And we allow `mCsmaCaEnabled = false` to be used with non-zero `mTxDelay` (when transmission is scheduled for a certain time), allowing transmission at the specified time without performing CCA.

Thoughts?

Reply:
I meant `mTxDelay` together with `mTxDelayBaseTime` are independent with `mCsmaCaEnabled`. I can add more text explain it.